src/components/productCard/ProductCard.tsx

```diff
@@ -20,7 +30,9 @@ const ProductCard = ({
         <img src={image} alt={title} />
       </div>
       <div
-        className="card-description"
+        className={`card-description ${
+          category === "Men's clothing" ? "men-background" : "women-background"
```

Comment:
@N-i-z One last thing, it's good to extract these string values as an `Enum` & use that for comparisons! That way we don't need to change the code if  we want to change the value later..

Look into `Enums` and create one for category & replace `category === "Men's clothing"` string value!